Gems/Atom/Tools/AtomToolsFramework/Code/Include/AtomToolsFramework/Application/AtomToolsApplication.h

```diff
@@ -32,8 +32,8 @@ namespace AtomToolsFramework
 {
     //! Base class for Atom tools to inherit from
     class AtomToolsApplication
-        : public AzFramework::Application
-        , public AzQtComponents::AzQtApplication
+        : public AzQtComponents::AzQtApplication
+        , public AzFramework::Application
```

Comment:
What does this ordering change do?

Reply:
I was experimenting with destruction order for the crash